src/tunnel.cc

```diff
@@ -1453,8 +1453,10 @@ TunnelStateData::startConnecting()
 void
 TunnelStateData::usePinned()
 {
-    Must(request);
+    Assure(request);
```

Comment:
Please undo this out-of-scope and distracting improvement:

```suggestion
    Must(request);
```

In the future, please avoid out-of-scope changes (by default).